code/game/objects/items/mail.dm

```diff
@@ -332,3 +334,166 @@
 /obj/item/paper/fluff/junkmail_generic/Initialize(mapload)
 	default_raw_text = pick(GLOB.junkmail_messages)
 	return ..()
+
+// =====================
+// Antag stuff down here
+// =====================
+
```

Comment:
Do not do ASCII art comments. Doc-comment `/obj/item/mail/traitor` instead (with just `/// Explain what it is here`)

Reply:
Uh, okay. I made it not for documentation purpose, but for an readability purpose. I will delete it.